vassal-app/src/main/java/VASSAL/tools/version/GitProperties.java

```diff
@@ -42,7 +43,7 @@ private void readGitProperties() {
         return;
       }
 
-      try (BufferedReader reader = new BufferedReader(new InputStreamReader(is))) {
+      try (BufferedReader reader = new BufferedReader(new InputStreamReader(is, StandardCharsets.UTF_8))) {
```

Comment:
Can we rely on git.properties always being UTF-8?

Reply:
Charset.defaultCharSet() should return 'windows-1252' on a windows system and 'utf-8' on anything else and should be used for writing generic text files.

Reply:
> Can we rely on git.properties always being UTF-8?

I think so, our template `git.properties` is UTF-8, the build happens on a UTF-8 system, and Maven uses Java underneath and should keep the encoding of this file during resource filtering.